jamjar/jamjar/concerts/models.py

```diff
@@ -13,7 +13,7 @@ class Concert(BaseModel):
 
     def make_graph(self):
         concert_id = self.id
-        concert_edges = Edge.objects.filter(video1__concert_id=concert_id).select_related('video1', 'video2')
+        concert_edges = Edge.objects.filter(video1__concert_id=concert_id, video2__concert_id=concert_id).select_related('video1', 'video2')
```

Comment:
yeah, AND is indeed what we want here. This code avoids matching videos across two different concerts. IT should probably be in a different branch but idk if it's a big deal